sdk/identity/identity/src/util/logging.ts

```diff
@@ -126,6 +127,7 @@ export function credentialLogger(title: string, log: AzureLogger = logger): Cred
   const credLogger = credentialLoggerInstance(title, undefined, log);
   return {
     ...credLogger,
+    parent: log,
```

Comment:
What does this change do? The PR description does not have anything regarding this.

Reply:
This is an internal change that allows us to test what the parent is logging.